docker-compose.yml

```diff
@@ -51,7 +51,7 @@ services:
     ports:
       - "8080:8080"
     networks:
-      - router_net
+      - hyperswitch_net
```

Comment:
Please revert this change, and replace `hyperswitch_net` with `router_net` everywhere else.